### PR TITLE
fix(issuer-api): Certificate events syncing

### PR DESCRIPTION
**Pull request by josipbagaric**
Fixes:
1. `TransferBatch` and `ClaimBatch` was listening to the wrong on-chain event
2. Remove double syncing certificates - only sync certificates when a blockchain event is detected
3. Sync certificates from an event instead of an internal `sync()` function
4. Wait for transactions to be mined before sending a confirmation

### CODE REVIEW

File: packages/traceability/issuer-api/src/pods/certificate/handlers/transfer-certificate.handler.ts
Comment (soanvig) [inline packages/traceability/issuer-api/src/pods/certificate/handlers/transfer-certificate.handler.ts]: What's that wait(1)? It looks creepy

Comment (josipbagaric) [reply packages/traceability/issuer-api/src/pods/certificate/handlers/transfer-certificate.handler.ts]: Wait for 1 transaction confirmation for the mined block.

Comment (soanvig) [reply packages/traceability/issuer-api/src/pods/certificate/handlers/transfer-certificate.handler.ts]: For a sec I thought it's like 1ms or something :D Thanks god

**Comment by soanvig** [comment]
I reviewed the changes, but those are too domain specific to approve them (for me, for now), so I leave it to @ioncreature

**Review by ioncreature — APPROVED** [review]

**Pull request merged**
